demisto_sdk/commands/generate_docs/readme_templates.py

```diff
@@ -0,0 +1,37 @@
+SYSLOG = (
+    "\n# <Product Name>\nThis pack includes Cortex XSIAM content.\n\n\n## Configuration on Server Side\n<Add "
```

Comment:
perhaps we should replace `<Product Name>` with `%%UPDATE_RN%%`, so that validations fail if users forget to fill it. wdyt?

Reply:
Might be a good idea.
@yasta5 WDYT? I used the templates you mentioned in the issue.

Reply:
Sounds like a good idea, though isn't RN stands for ReleaseNotes? We can maybe put <%%UPDATE_ReadME%%>

Reply:
We will use %%UPDATE_README%% and it will fail the readme validation if it isn't removed.